README.md

```diff
@@ -1 +1,15 @@
 Urdf & Gazebo files for the ABB YuMi (IRB 14000) robot
+
+### Dependencies
+This package depends on ros-industrial which is not released on kinetic yet. So skip step 2 if you have already install ros-industrial.
+
+#### Step 1
+Install all of these:
+
+> sudo apt-get install ros-kinetic-control-toolbox ros-kinetic-controller-interface ros-kinetic-controller-manager ros-kinetic-joint-limits-interface ros-kinetic-transmission-interface ros-kinetic-moveit-core ros-kinetic-moveit-planners ros-kinetic-moveit-ros-planning 
+
+#### Step 2
+Install ros-industrial from source. 
```

Comment:
note: if this will run on a real yumi (not in simulation), you need to pull our fork of ros-industrial from https://github.com/OrebroUniversity/industrial_core.git